CONTRIBUTING.md

```diff
@@ -1,5 +1,15 @@
 # Contributing
 
+## Before
+
+Before creating a Pull Request, please contact us at [issues](https://github.com/moeru-ai/xsai/issues) or [discussions](https://github.com/moeru-ai/xsai/discussions) to confirm that the feature in question is one we need.
+
+When creating a Pull Request, please make sure you have read and agreed to it:
+
```

Comment:
```suggestion
- We will do our best to try to explain and discuss the possible implementation approaches as much detailed as possible before modifications and Pull Request state changes, but still, sometimes it may be much more efficient for us to directly use actual code to address our opinions.
```